### PR TITLE
Adds section to README on configuring kubelet config

### LINKED ISSUE #672: how to apply registryPullQPS in files/kubelet-config.json
how to apply registryPullQPS in files/kubelet-config.json 
Is this right? 
Then, if right this configuration, how to build ec2 ami with to modify applied kubelet-config.json 

{
  "kind": "KubeletConfiguration",
  "apiVersion": "kubelet.config.k8s.io/v1beta1",
  "address": "0.0.0.0",
  "authentication": {
    "anonymous": {
      "enabled": false
    },
    "webhook": {
      "cacheTTL": "2m0s",
      "enabled": true
    },
    "x509": {
      "clientCAFile": "/etc/kubernetes/pki/ca.crt"
    }
  },
  "authorization": {
    "mode": "Webhook",
    "webhook": {
      "cacheAuthorizedTTL": "5m0s",
      "cacheUnauthorizedTTL": "30s"
    }
  },
  "clusterDomain": "cluster.local",
  "hairpinMode": "hairpin-veth",
  "readOnlyPort": 0,
  "cgroupDriver": "cgroupfs",
  "cgroupRoot": "/",
  "featureGates": {
    "RotateKubeletServerCertificate": true
  },
  "protectKernelDefaults": true,
  "serializeImagePulls": false,
  "serverTLSBootstrap": true,
  "tlsCipherSuites": ["TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256", "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256", "TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305", "TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384", "TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305", "TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384", "TLS_RSA_WITH_AES_256_GCM_SHA384", "TLS_RSA_WITH_AES_128_GCM_SHA256"],
  "registryPullQPS": 0,
  "registryBurst": 100
}

### PR DESCRIPTION
**Pull request by mmerkes**
**Description of changes:**

Adds section to README on configuring kubelet config. Includes information on setting the `--kubelet-extra-args` flag with the bootstrap script or configuration the kubelet config file.

Resolves #672 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Testing Done**

Verified instructions as is on a cluster.

*[See this guide for recommended testing for PRs.](https://github.com/awslabs/amazon-eks-ami/blob/master/CONTRIBUTING.md#testing-changes) Some tests may not apply. Completing tests and providing additional validation steps are not required, but it is recommended and may reduce review time and time to merge.*

### CODE REVIEW

**Comment by cartermckinnon** [comment]
Could this go in a `doc/` directory instead of the README? We could add a `More Info` section or something that links to these longer-form topics.

**Comment by mmerkes** [comment]
> Could this go in a `doc/` directory instead of the README? We could add a `More Info` section or something that links to these longer-form topics.

I had considered doing something like that, but I decided that was overkill for this one section. However, as we add additional documentation to the repo, I think we should do something like that.

File: README.md [hunk 1]
```diff
@@ -109,6 +106,58 @@ By default, the `amazon-eks-ami` uses a [source_ami_filter](https://github.com/a
...
+## Customizing Kubelet Config
+
+In some cases, customers may want to customize the [kubelet configuration](https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration) on their nodes, and there are two mechanisms to do that with the EKS Optimized AMI.
+
+**Set the "--kubelet-extra-args" flag when invoking bootstrap.sh**
+
+`bootstrap.sh`, the script that bootstraps nodes when using the EKS Optimized AMI, supports a flag called `--kubelet-extra-args` that allows you to pass in additional `kubelet` configuration. If you invoke the bootstrap script yourself (self-managed nodegroups or EKS managed nodegroups with custom AMIs), you can use that to customize your configuration. For example, you can use something like the following in your userdata:
+
+```
+/etc/eks/bootstrap.sh my-cluster --kubelet-extra-args '--registry-qps=20 --registry-burst=40'
```
Comment (cartermckinnon) [inline README.md:118]: nit: add another common arg folks set like log verbosity

**Comment by cartermckinnon** [comment]
The big section about kubelet binaries could be moved to it's own file too, IMO. But yeah, we can always do it later

**Review by cartermckinnon — APPROVED** [review]

**Pull request merged**